packages/stems/src/components/Button/Button.module.css

```diff
@@ -352,6 +352,7 @@
   cursor: default;
   transform: none;
   border: none;
+  pointer-events: none;
```

Comment:
oh, this is a spooky. its just for disabled buttons so probably ok, but want to make sure that this is given due consideration. lots of buttons

Reply:
oh interesting. ok sounds good to me. this may cause issues with my disabled toast thing, but I'll figure it out if needed 👍

Reply:
oh lets chat about that actually